### PR TITLE
chore: v0.0.5

### PR DESCRIPTION
**Pull request by maherjaafar**
## Status

**READY**

## Breaking Changes

NO

## Description

Bump version 0.0.5 to update the package in pub.dev

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore
- [ ] 🧪 Testing

### CODE REVIEW

File: pubspec.yaml
Comment (felangel) [inline pubspec.yaml:7]: Why are we bumping the version? We should wait until new features have been merged before bumping the version and publishing to pub.dev

Comment (maherjaafar) [reply pubspec.yaml:7]: [This refactor](https://github.com/felangel/flow_builder/commit/33ac8afab3783c243bcb4d47846222efa9afab35) is already done.

Comment (felangel) [reply pubspec.yaml:7]: In that case, can you also update the CHANGELOG.md?

Comment (maherjaafar) [reply pubspec.yaml:7]: Sure!

Comment (maherjaafar) [reply pubspec.yaml:7]: Done ✅

File: CHANGELOG.md
Comment (felangel) [inline CHANGELOG.md:3]:
```suggestion
- fix: Intellisense on update to include state argument [#64](https://github.com/felangel/flow_builder/issues/64)
```

File: CHANGELOG.md
Comment (felangel) [inline CHANGELOG.md:3]:
```suggestion
- fix: intellisense on update to include state argument [#64](https://github.com/felangel/flow_builder/issues/64)
```

**Review by felangel — APPROVED** [review]
LGTM 🚀🎉

**Comment by maherjaafar** [comment]
@felangel I've missed to add the v0.0.5 **tag**

**Comment by felangel** [comment]
> @felangel I've missed to add the v0.0.5 **tag**

I’ll tag and publish shortly, no worries!

**Comment by maherjaafar** [comment]
> > @felangel I've missed to add the v0.0.5 **tag**
> 
> I’ll tag and publish shortly, no worries!

Thanks! 👍🏼

**Pull request merged**
